std_msgs/Header header

string leader_name
string text_info
geometry_msgs/Polygon waypoints